themes/default/content/docs/concepts/inputs-outputs/lifting.md

```diff
@@ -0,0 +1,316 @@
+---
+title_tag: "Accessing Nested Properties with Lifting | Inputs and Outputs"
+meta_desc: "Learn how to access nested output values using the lifting method in Pulumi."
+title: Accessing nested properties with Lifting
+h1: Accessing nested properties with Lifting
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  concepts:
+    weight: 4
+    parent: inputs-outputs
+---
+
+While often, Outputs return asynchronous values that wrap primitive types like strings or integers, sometimes an output has an object with deeply nested values. These properties need to be passed to other inputs as well.
+
+For example, to read a domain record from an ACM certificate, you need to access the domain validation options, which returns an array. Because that value is an output, we would normally need to use {{< pulumi-apply >}}:
+
+{{< chooser language "javascript,typescript,python,go,csharp,java" >}}
```

Comment:
Love the addition of the explanations in Go regarding the use of `Index()` and `Elem()`, again a miss in the docs previously.